lib/dashboard/DefaultDashboardFactory.ts

```diff
@@ -58,22 +58,30 @@ export interface MonitoringDashboardsProps {
   /**
    * Flag indicating whether the default dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
-  readonly createDashboard: boolean;
+  readonly createDashboard?: boolean;
   /**
    * Flag indicating whether the summary dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
   readonly createSummaryDashboard: boolean;
   /**
    * Flag indicating whether the alarm dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
   readonly createAlarmDashboard: boolean;
```

Comment:
As mentioned above by Eugene, this should be optional too since we now have a default value.

Reply:
Sure - it was a miss. Fixed in next rev.